sysl2/lib/lang/java.sysl

```diff
@@ -0,0 +1,889 @@
+# sysl.lang.java
```

Comment:
This is somewhat of a duplicate from the docs: https://github.com/anz-bank/sysl/blob/master/docs/ideas/grammar.md - can you please delete the docs section and put a link to this file there?
It seems like the grammar's grammar is loosely based on 
https://github.com/anz-bank/sysl/blob/sysl2/sysl2/proto/grammar.proto (which hasn't been reviewed).
This seems to be quite a core concept, so can we please have some documentation on this grammar - the `proto` file is not the best for comprehension.
You might want to use the [grammar.md](https://github.com/anzdaddy/sysl/blob/master/docs/ideas/grammar.md) as a starting point

Reply:
Agreed we need docs #123, but shouldn't block this PR.

Reply:
Well, I'd prefer to have things nice, clean and complete - remember the [broken window theory](https://blog.codinghorror.com/the-broken-window-theory/) ?

Otherwise we run risk of having heaps of Issues for outstanding documentation and/or tests.
It doesn't need to be much, probably pulling out the comments from the `proto` file and/or updating [grammar.md](https://github.com/anzdaddy/sysl/blob/master/docs/ideas/grammar.md) would be sufficient and I bet it's done in less than 5 minutes. It would set a great example.

If you absolutely insist that this cannot be done now and/or is likely to change so much in the imminent future that it doesn't warrant the effort, please let me know.